src/main/java/com/google/sps/data/Player.java

```diff
@@ -31,9 +38,11 @@ public Player(
       String id,
       String imageID,
       String currentPageID,
-      int experiencePoints) {
+      int experiencePoints,
+      int promotionThreshold) {
     this(displayName, email, id, imageID, currentPageID);
     this.experiencePoints = experiencePoints;
+    this.promotionThreshold = promotionThreshold;
```

Comment:
Why should the promotion threshold be player specific instead of game stage specific? Should it maybe be harder to get promoted with each increasing game stage level?

Reply:
maybe we can make every gameStage have it's own threshold parameter rather than the player

Reply:
The threshold and how much the threshold increases are different. I'll need a new servlet/ have to modify another servlet to get the game stage level to find out how much to increment the threshold, which shouldn't be a problem. 

In storing the promotion threshold, I only have to fetch it/update it once each gameStage.html load. If I were to instead store how many points have been earned this game stage, it would be updated and stored every time a player earns an experience point, which would slow down the page response. I'd also have two experience point counter properties for each player which would get confusing.